platform/feature-api/api/pod-api/src/default.ts

```diff
@@ -160,19 +160,23 @@ export class DefaultPod implements Pod {
      */
     get endpoint(): Endpoint {
         return {
-            send(
-                endpointId: string,
-                payload: string,
-                contentType?: string,
-                authToken?: string
-            ): Promise<void> {
+            send(request: {
+                endpointId: string;
```

Comment:
Defining an interface (or directly a class) would avoid repetition of the parameters here.